src/patternfly/components/DataToolbar/data-toolbar.scss

```diff
@@ -6,22 +6,28 @@ $pf-c-data-toolbar--spacer-map: build-spacer-map("none", "sm", "md", "lg");
 // Data toolbar
 .pf-c-data-toolbar {
   --pf-c-data-toolbar--BackgroundColor: var(--pf-global--BackgroundColor--100);
-  --pf-c-data-toolbar--GridGap: var(--pf-global--gutter);
+  --pf-c-data-toolbar--Gap: var(--pf-global--gutter);
```

Comment:
should this var: `--pf-c-data-toolbar--Gap` be named `--pf-c-data-toolbar--Gutter`

Reply:
Using this as a reference, seems like we're moving toward `gap` instead of `gutter`: https://github.com/patternfly/patternfly-next/issues/2336. I could use `row-gap` instead.

Reply:
I think `Gap` works since the primary function is to define the `gap` property, and isn't used as a concept like a gutter.

Reply:
thats cool with me